products/models.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+from users.models import TimeStampedModel
+
+class Product(TimeStampedModel):
+    product_name    = models.CharField(max_length=15)
+    product_content = models.CharField(max_length=300)
+    people          = models.IntegerField()
+    cook_time       = models.IntegerField()
+    setting_time    = models.IntegerField()
+    selling_price   = models.DecimalField(max_digits=9, decimal_places=2)
+    discount_price  = models.DecimalField(max_digits=9, decimal_places=2, null=True)
+
+    class Meta:
+        db_table = 'products'
+
+class ProductImage(models.Model):
+    product_image = models.CharField(max_length=100)
```

Comment:
상품 이미지에 대한 데이터는 어떤걸로 저장하나요? 만약 보통의 경우와 같이 URL로 저장하신다면 max_length를 지금보다 더 넉넉히 지정해주는게 좋을 것 같습니다.
```suggestion
    product_image = models.CharField(max_length=300)
```

Reply:
네 300으로 수정했습니다